rasa/cli/data.py

```diff
@@ -98,5 +98,7 @@ def validate_files(args):
         domain_path=args.domain, training_data_paths=args.data
     )
 
-    validator = loop.run_until_complete(Validator.from_importer(file_importer))
+    validator = loop.run_until_complete(
```

Comment:
is there any reason not to directly configure the logger in here instead of in init of the `Validator`?